model/font.go

```diff
@@ -420,16 +420,26 @@ func (font *PdfFont) BytesToCharcodes(data []byte) []textencoding.CharCode {
 	return charcodes
 }
 
-// CharcodesToUnicodeWithStats is identical to CharcodesToUnicode except returns more statistical
+// CharcodesToUnicodeWithStats is identical to CharcodesToUnicode except it returns more statistical
 // information about hits and misses from the reverse mapping process.
-func (font *PdfFont) CharcodesToUnicodeWithStats(charcodes []textencoding.CharCode) (runelist []rune, numHits, numMisses int) {
+// NOTE: The number of runes returned may be greater than the number of charcodes.
+// TODO(peterwilliams97): Deprecate in v4 and use only CharcodesToStrings()
+func (font *PdfFont) CharcodesToUnicodeWithStats(charcodes []textencoding.CharCode) (text string, numHits, numMisses int) {
```

Comment:
This change we cannot do now, since it would be a breaking change.
See our proposed compatibility guidelines https://github.com/unidoc/unipdf/issues/345
Under Major changes
- Changing an exported function/method signature

Reply:
We should leave this function as was, and with the deprecation warning.
```
func (font *PdfFont) CharcodesToUnicodeWithStats(charcodes []textencoding.CharCode) (runelist []rune, numHits, numMisses int) {
```

Reply:
Done